{"uuid":"3ee0b367-9039-4a12-9bb2-f3c8036ecffe","historyId":"a94cda9cc9f06793a63a9aa174afe3aa","fullName":"io.cucumber.testng.AbstractTestNGCucumberTests.runScenario","labels":[{"name":"package","value":"testRunner.DsalgoRunner"},{"name":"testClass","value":"testRunner.DsalgoRunner"},{"name":"testMethod","value":"runScenario"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"testRunner.DsalgoRunner"},{"name":"host","value":"LAPTOP-2KQV9F30"},{"name":"thread","value":"21676@LAPTOP-2KQV9F30.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Runs Cucumber Scenarios","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"BUG: Expected message 'Submission Successful' but got: 'Some Tests failed. Please review code' expected [true] but found [false]","trace":"java.lang.AssertionError: BUG: Expected message 'Submission Successful' but got: 'Some Tests failed. Please review code' expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\r\n\tat org.testng.Assert.assertTrue(Assert.java:57)\r\n\tat stepDefinitions.ArraysStep.the_user_gets_success_message_submission_successful_in_the_console(ArraysStep.java:320)\r\n\tat âœ½.The user gets success message Submission successful in the console(file:///C:/Workspace/Ds-Algo2025/src/test/resources/Feature/Arrays.feature:81)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg1","value":"\"Arrays PracticeQuestion\""},{"name":"arg0","value":"\"Verify that user receives Success message on submitting valid python code for \"Search the array\" question\""}],"start":1746302554997,"stop":1746302559611}